semcore/d3-chart/src/a11y/serialize.ts

```diff
@@ -200,7 +200,7 @@ export const serialize = (
       const mainSummary: string = intl.formatMessage(
         { id: 'time-series-general-trend' },
         {
-          dataKey: entitiesCount !== 1 ? dataKey : '',
+          dataKey: entitiesCount !== 1 ? primaryTrend.label ?? dataKey : '',
```

Comment:
In `insights.ts` you have added `.displayValue` but here you use `.label`. Is it correct?

Reply:
yes, this is another object and `label` here is a value of `displayValue` from `insights`